README.md

```diff
@@ -1,70 +1,58 @@
-# Getting Started with Create React App
+# Country-Facts
+Microverse Module 03 Project: This educational project is to create a React Country-Facts Application that allows the user to display various metric information about a given selected Country from an API.
+ 
```

Comment:
- [ ] I'm not able to find a link to your presentation video please add it before requesting another review.